collectors/tc.plugin/plugin_tc.c

```diff
@@ -12,60 +12,48 @@
 #define TC_LINE_MAX 1024
 
 struct tc_class {
-    avl_t avl;
+    STRING *id;
+    STRING *name;
+    STRING *leafid;
+    STRING *parentid;
+
+    bool hasparent:1;
+    bool isleaf:1;
+    bool isqdisc:1;
+    bool render:1;
+    bool name_updated:1;
+    bool updated:1;
```

Comment:
The only portable, non-implementation defined type that makes sense to use as a bitfield is a signed integer. Just use plain bools, here.

Reply:
thanks! solved it.